apps/web/pages/api/book/event.ts

```diff
@@ -290,6 +291,46 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return g;
   });
 
+  // For seats, if the booking already exists then we want to add the new attendee to the existing booking
+  if (reqBody.bookingId) {
```

Comment:
If there is a bookingId that means the booking is already created and we are adding more attendees. No need to go through the rest of the endpoint to create the booking